### DISCUSSION TITLE
Why 'gz.org_opengrok' file is created after indexing?

### DISCUSSION THREAD

**Discussion by js3416** [oracle/opengrok#3978] [General]
Hi team, 

I found 'gz.org_opengrok' files are created after index on my project. 
-rwxrwxrwx   1 xxxx users    92 Jun 21 21:58 Makefile
-rw-rw-rw-   1 xxxx users   374 Jun 27 15:18 Makefile.gz.org_opengrok
-rw-rw-rw-   1 xxxx users    20 Jun 27 15:18 Makefile.gz.org_opengrok.gz.org_opengrok

Why these files are existed?

I made symbolic link xref file as below. It could be a cuase?

/local2/mnt/opengrok/data$ ls -l
lrwxrwxrwx  1 xxxx users    26 Jun 15 18:42 xref -> /local2/mnt/opengrok/src/

Thanks.

**Comment by vladak** [comment]
what OpenGrok version is this ?

**Reply by js3416** [reply]
The version is 1.5.11.
